caper/cromwell_workflow_monitor.py

```diff
@@ -42,6 +43,9 @@ def parse(self, line, workflow_status_map):
                 Workflow's string ID.
             status:
                 New status after transition.
+            auto_write_metadata:
+                For this status transition metadataJSON file should be written
+                on workflow's root output directory.
         """
         r = re.findall(self._regex, line)
         if len(r) > 0:
```

Comment:
```suggestion
        if r:
```
Is more pythonic. If there are no matches `r=[]`, which is "Falsy".

Reply:
Yep I agree. Fixed as you suggested.